lib/httpMethods.js

```diff
@@ -25,7 +25,9 @@ const bodyMethods = new Set([
   'MKCOL',
   'PROPFIND',
   'PROPPATCH',
-  'SEARCH'
+  'SEARCH',
+  'MKCALENDAR',
+  'REPORT'
```

Comment:
This is normally in alphabetical order

Also just realized 'MKCOL' should be above 'MOVE' for the same reason